namespaces/live-1.cloud-platform.service.justice.gov.uk/formbuilder-submit-product-production/circleci-service-account.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: circleci-formbuilder-submit-product-production
+  namespace: formbuilder-submit-product-production
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: circleci-formbuilder-submit-product-production
+  namespace: formbuilder-submit-product-production
+subjects:
+- kind: ServiceAccount
+  name: circleci-formbuilder-submit-product-production
+  namespace: formbuilder-submit-product-production
+roleRef:
+  kind: ClusterRole
```

Comment:
Hi 
Can you please create a role, similar to below, instead of using `ClusterRole`

https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/c100-application-production/serviceaccount-circleci.yaml#L8

Reply:
done